starter_code/models/movie.js

```diff
@@ -0,0 +1,33 @@
+const mongoose = require("mongoose");
+
+const Schema = mongoose.Schema;
+
+const movieSchema = new Schema (
+  {
+    title: {
+      type: String,
+      required: true,
+    },
+    director: {
+      type: String,
+    },
+    stars: {
```

Comment:
for stars and showtimes all you need to put is ```type : Array ``` . You can check this in your seeds.js